angularSTEP/src/app/profile-card/profile-card.component.spec.ts

```diff
@@ -30,4 +30,8 @@ describe('ProfileCardComponent', () => {
         expect(component.picUrl).toBeInstanceOf(String);
         expect(component.username).toBeInstanceOf(String);
     });
+
+    it('should not change', () =>{
```

Comment:
'should not change' under what circumstances? What about coverage for when it should change? It'd be better to test both of these behaviors through the component constructor rather as well - calling `shortenDisplayName` directly is only good for testing its own behavior - not the behavior of the component

Reply:
Im having trouble testing this function. it doesn't take in input or anything. Do you have a suggestion for how to test this?

Reply:
I figure it'd involve injecting a specific UserData model into the constructed component, but I'm not familiar enough with testing Angular to say exactly how to do this. Then later the test would just assert that `displayName` is (not) shortened as expected